app/components/primer/alpha/select_panel.html.erb

```diff
@@ -32,20 +41,12 @@
                   position: :absolute
                 }
               )) %>
+              <%= render(Primer::Alpha::ActionList::Divider.new(mb: 0)) %>
```

Comment:
Hmm the `Divider` component isn't really designed to be used like this. Can we write a bit of CSS instead?

Reply:
Doesn't the header have a divider param? The `Dialog` has `show_divider`

Reply:
Oh dang it does! https://github.com/primer/view_components/blob/6348ede37a08d9b325aa24bd30156bd9a8377a08/app/components/primer/alpha/dialog/header.rb#L56